### PR TITLE
feat: DPS prep changes outlined in #1

### PR DESCRIPTION
**Pull request by wildintellect**
Getting algorithm ready for DPS. Working out issue around R package installation. See #1

### CODE REVIEW

**Comment by wildintellect** [comment]
@chuckwondo can you look this over to see if I missed anything. @nehahunka I know we're going to need to test this a bunch to see if it works. If you can prep a notebook to call a job with the right parameters that would be helpful.

File: NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh [hunk 1]
```diff
@@ -1,16 +1,8 @@
 #!/bin/bash
 set -x
 basedir=$( cd "$(dirname "$0")" ; pwd -P )
-conda env update -n base --solver=libmamba -f ${basedir}/env_main_ADE.yaml
+# Install dependencies
+conda env update -n r -f ${basedir}/env_main_ADE.yaml
```
Comment (chuckwondo) [inline NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh]: minor: for consistency with the `conda run` command that follows, consider spelling out `--name`:
```suggestion
conda env update --name r -f ${basedir}/env_main_ADE.yaml
```

File: NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh [hunk 2]
```diff
@@ -1,16 +1,8 @@
...
-# conda install conda-forge::r-inlabru --yes
-# conda install conda-forge::r-terra --yes
-# conda install conda-forge::r-dplyr --yes
-# conda install conda-forge::r-spdep --yes
-# conda install conda-forge::r-sf --yes
-# conda install conda-forge::r-fields --yes
-# conda install conda-forge::r-Matrix --yes
\ No newline at end of file
+# Install INLA
+conda run --name r --no-capture-output Rscript ${basedir}/install.R
```
Comment (chuckwondo) [inline NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh]: Where is `install.R`? (Also, add a newline to the end of this file)

File: NASA_CMS_2023/Mexico/DPS_algorithm/run_GMBpredictions_MEX.sh [hunk 3]
```diff
@@ -16,4 +16,4 @@ Projects="input/Projects.gpkg"
 
 OUTPUTDIR="${PWD}/output"
 
-Rscript ${basedir}/GMB_Mexico.R ${output_csv} ${mexico} ${ccirast} ${heirast} ${data} ${INLA_model_fit_v2} ${MGN2020_INEGI_Urban_Mex_mask_DISS} ${All_Products_Comp_Over30_Binary_6933} ${Projects} 
\ No newline at end of file
+conda run --name r --no-capture-output Rscript ${basedir}/GMB_Mexico.R ${output_csv} ${mexico} ${ccirast} ${heirast} ${data} ${INLA_model_fit_v2} ${MGN2020_INEGI_Urban_Mex_mask_DISS} ${All_Products_Comp_Over30_Binary_6933} ${Projects}
```
Comment (chuckwondo) [inline NASA_CMS_2023/Mexico/DPS_algorithm/run_GMBpredictions_MEX.sh]: minor: add a newline to the end of this file

File: NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh (hunk 2 above)
Comment (wildintellect) [reply NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh]: oops forgot to add the file

File: NASA_CMS_2023/Mexico/DPS_algorithm/run_GMBpredictions_MEX.sh (hunk 3 above)
Comment (wildintellect) [reply NASA_CMS_2023/Mexico/DPS_algorithm/run_GMBpredictions_MEX.sh]: also added to the build_command.sh

File: NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh (hunk 2 above)
Comment (wildintellect) [reply NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh]: Added now

File: NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh (hunk 1 above)
Comment (wildintellect) [reply NASA_CMS_2023/Mexico/DPS_algorithm/build_command_main_ADE.sh]: fixed

**Review by chuckwondo — APPROVED** [review]
Looks good as far as I can tell. GitHub is indicating that there's some conflict with the `main` branch, but I can't see what it is.

**Pull request merged**